### ISSUE TITLE
2 kiosk on 2 monitors?

### ISSUE THREAD

**Issue by SaymonDzen** [grafana/grafana-kiosk#52]
I decided to give up raspberries because she stopped coping with our boards and took a more powerful computer and connected two monitors with different boards to it. But I can't figure out how to run the third instance on another screen. In ubuntu 20.04, both monitors display: 0
Please help!

**Comment by briangann** [comment]
you should be about to tell the kiosk which monitor to use by exporting/setting the env variable `DISPLAY` while launching the kiosk.

It defaults to `DISPLAY=:0.0`

**Comment by SaymonDzen** [comment]
Hi, brianann!
I wrote both physical monitors are defined as a single space and with DISPLAY 0.0 you cannot specify a physical monitor. If it were so simple, I would not ask for help.

**Comment by RyuunosukeDS3** [comment]
I think what you want is https://github.com/grafana/grafana-kiosk/issues/39
I've tried to implement it myself as i commented there, but it didn't work very well...

**Comment by briangann** [comment]
i'll get a pi4 and get this working, sounds very useful.  my kiosks are pi3 with single display, no reason this can't be implemented.

**Comment by SaymonDzen** [comment]
I found a workaround https://vladvasiliu.com/post/20180807-1626-dual_screen_kiosk_ubuntu/ Maybe someone will be useful Maybe someone will be useful

**Comment by pconwell** [comment]
I see this issue was closed with PR #54, but I don't see how to implement it. I see on the main repo page it says "Can specify where to start kiosk for multiple displays", but I don't understand _how_ to specify which display and I don't see any other references in the documentation.

I have the first playlist working correctly on the first display. How do I get the second playlist to play on the second display?

Here is my systemctl service:

```
[Unit]
Description=Grafana Kiosk
Documentation=https://github.com/grafana/grafana-kiosk
Documentation=https://grafana.com/blog/2019/05/02/grafana-tutorial-how-to-create-kiosks-to-display-dashboards-on-a-tv
After=network.target
After=graphical.target
After=multi-user.target

[Service]
Type=idle
ExecStartPre=/bin/sleep 30
User=pi
Environment="DISPLAY=:0"
Environment="XAUTHORITY=/home/pi/.Xauthority"
ExecStart=/usr/bin/grafana-kiosk -URL=[URL] -login-method=local -username=[USER] -password=[PASSWRD] -playlists=true -autofit=false
Restart=always

[Install]
WantedBy=graphical.target
```
